internal/action/env.go

```diff
@@ -20,6 +20,12 @@ func (s *Action) Env(c *cli.Context) error {
 	name := c.Args().First()
 	args := c.Args().Tail()
 
+	// FIXME? I have the feeling that this test should be done when parsing the command
```

Comment:
Unsure if this is the right place for the fix, but I don't know urfave...

Reply:
This looks correct to me.
What are you unsure about?

Reply:
I wasn't clear :-)
I meant: one could argue that this error should be detected and handled directly by urfave when parsing the command-line, in the same spirit as omitting a required argument or flag. But I am not familiar with urfave, so I don't know if this is possible at all.

If it is fine for you to keep the check here, then it is fine for me also :-)

Reply:
Maybe there is a way to do that, but I'm not aware of it.
So let's keep as is.

Reply:
removed comment